library/src/main/java/com/afollestad/vvalidator/assertion/input/InputAssertions.kt

```diff
@@ -318,6 +327,25 @@ sealed class InputAssertions {
 
     override fun defaultDescription() = "must match regex \"$regexString\""
   }
+
+  /** @author Suu SoJeat (@suusojeat) */
+  class ComparesAssertion internal constructor(
+    private val targetView: EditText
+  ) : Assertion<EditText, ComparesAssertion>() {
+    private var ignoreCase: Boolean = false
+
+    /** Case is ignored when comparing the two input text. */
+    fun ignoreCase(): ComparesAssertion {
+      ignoreCase = true
+      return this
+    }
+
+    override fun isValid(view: EditText): Boolean {
+      return view.text().compareTo(targetView.text(), ignoreCase = ignoreCase) == 0
```

Comment:
This is the same as `equalsIgnoreCase(...)` 🙂

Reply:
I'm sorry. I don't quite get what you meant.

Reply:
Sorry, i was busy with exams in the past week.